OrderManagement/src/main/java/com/flmfoods/model/Order.java

```diff
@@ -0,0 +1,36 @@
+package com.flmfoods.model;
+
+import java.util.List;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name="orders")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Order {
+
+	private long orderId;
+	private String status;
+	private double orderPrice;
+	@ManyToOne
+	@JoinColumn(name="user_id")
+	private User user;
```

Comment:
```suggestion
	private long userId;
```
We dont have a User class here, so we just add userId, no mappings required